tensorboard/webapp/runs/store/runs_types.ts

```diff
@@ -68,10 +68,17 @@ export interface RunsDataState {
   runMetadata: Record<RunId, Run>;
   runsLoadState: Record<ExperimentId, LoadState>;
   hparamAndMetricSpec: ExperimentToHparamAndMetricSpec;
-  // Run selection is tied to a list of experimentIds which is somewhat related
-  // to route but not strictly. For instance, if we want to render the
-  // run-selector in both experiment list and dashboard routes sharing the
-  // state, they need to share a key.
+  /**
+   * Map from a 'key' to a `RunId` to boolean whether the run is selected.
+   * The 'key' is a serialization of multiple experiment IDs.
+   *
+   * Run selection is tied to a list of experimentIds which is somewhat related
+   * to route but not strictly. For instance, if we want to render the
+   * run-selector in both experiment list and dashboard routes sharing the
+   * state, they need to share a key.
+   *
+   * TODO: this belongs in UI state not data state.
```

Comment:
Avoid a TODO without a name of a bug. Either put your name or just say "hack".

Reply:
Done.